### PR TITLE
removing the helm chart

### PR DESCRIPTION
**Pull request by arshamparity**
According to the conversation on https://github.com/paritytech/substrate-telemetry/pull/368, a basic Helm chart has been created and pushed to the [Parity's public helm repository](https://github.com/paritytech/helm-charts/tree/main/charts/substrate-telemetry) and Parity's custom values file has been moved to a private repository([MR](https://gitlab.parity.io/parity/infrastructure/cloud-infra/-/merge_requests/585)).

cc @chevdor

### CODE REVIEW

**Comment by arshamparity** [comment]
https://github.com/paritytech/substrate-telemetry/issues/370

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -52,7 +44,8 @@ stages:
         --set image.backend.tag="${CI_COMMIT_SHORT_SHA}"
         --set image.frontend.repository="${CONTAINER_REGISTRY}/${FRONTEND_CONTAINER_REPO}"
         --set image.frontend.tag="${CI_COMMIT_SHORT_SHA}"
-        $KUBE_NAMESPACE ./helm/
+        --values $HELM_VALUES_FILE
+        $KUBE_NAMESPACE parity/substrate-telemetry
```
Comment (PierreBesson) [inline .gitlab-ci.yml:48]: Are you using not specifying the version of the chart on purpose ?

**Review by chevdor — APPROVED** [review]
LGTM

File: .gitlab-ci.yml (hunk 1 above)
Comment (arshamparity) [reply .gitlab-ci.yml:45]: `--version` has been added to the `helm upgrade` command and the sed commands which were used to modify `chart.yml` have been removed.

**Review by PierreBesson — APPROVED** [review]

**Review by PierreBesson — APPROVED** [review]

**Pull request merged**